content/en/blog/indsutry-academic-ospo.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Why do universities and enterprises care about open source collaboration?"
+author: anajs
+date: 2025-09-11
+---
+
+This summary is based on the panel discussion [“Open Source and Academic Collaboration: How OSPOs Make It Work”](https://youtu.be/N66M7UqWiyw?si=fxpK4gtcrLZikjog) recorded at an Open Source Summit. This panel continued a [discussion started at a previous OSSummit](https://curioss.org/news/nov24-industry-partnerships/) and that the group hoped 
+to identify trends, challenges, and opportunities for university–industry collaboration around open source.
+
+![Figure_1](https://github.com/user-attachments/assets/a7f426c5-f651-4f19-b7a3-69a3b46345b3)
+
+## Motivations from Industry
+
+**Leslie Hawthorne** of Red Hat explained that her company has a long history of working with academic institutions on open-source projects because it helps with talent development and product adoption. She described Red Hat’s goal of getting students excited about open source early on:  
+“I get to do a lot of really cool stuff with open source projects, including things like collaborating with various academic institutions.” From an industry perspective, hiring students who already contribute to open source makes onboarding easier: “We are a software engineering organization. 
+We would like to hire intelligent, competent people. If they’re already working in open source projects, obviously it’s easier.”  
+
+**Cornelius Schumacher**, open source steward at Deutsche Bahn’s IT arm, said that open source is not the company’s core business but a means to an end: “We invest in collaboration, open source is for us a means to an end; we are in the business of making trains run.” Their motivation for 
+partnering with universities is to advance research and logistics; open source provides an effective way to collaborate on software when needed.  
+
+## Motivations from Academia
+
+**Ying Wang**, Technology and Licensing Manager at ETH Zurich, described how their academic OSPO started by surveying researchers and students across disciplines. The group learned that university interests—*“freedom of publication, validation of the data and the results by peers”*—align well 
+with open-source values. At the same time, universities need funding and job prospects for students, while companies need skilled contributors; hence, collaboration can serve both sides. She also noted that universities face unique hurdles when contributing to corporate open-source projects 
+because contributor licence agreements (CLAs) often require signatures from the entire institution—something impractical for large universities. She suggested companies provide flexible CLA options.  
+
+**Chris Hoeppler** of Bosch Research/OSPO added that universities now need more than publication rights; they must share code and data: “OSPOs help people figure out those different aspects and the different needs of both sides.” Open collaboration and reproducibility have become expectations. 
```

Comment:
I've recently changed positions within Bosch and am not directly affiliated with Bosch Research any more.
```suggestion
**Chris Hoeppler** of the Bosch OSPO added that universities now need more than publication rights; they must share code and data: “OSPOs help people figure out those different aspects and the different needs of both sides.” Open collaboration and reproducibility have become expectations. 
```